packages/react-utilities/src/compose/getSlots.ts

```diff
@@ -1,8 +1,87 @@
 import * as React from 'react';
 
-import { getNativeElementProps, omit } from '../utils/index';
-import { GenericDictionary } from './types';
+import { ComponentState, ShorthandRenderFunction, SlotPropsRecord } from './types';
 import { nullRender } from './nullRender';
+import { getNativeElementProps, omit } from '../utils/index';
+
+function getSlot(
+  component: React.ElementType | undefined,
+  alternative: keyof JSX.IntrinsicElements | undefined,
+  defaultValue: React.ElementType,
```

Comment:
```suggestion
```

According to the spec, we should render `div` by default, do we need to have it configurable?
https://github.com/microsoft/fluentui/blob/master/specs/Slots.md#slots-components

Reply:
According to previous implementation of getSlots, it's `div` for root and `span` for slots